package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ux-pattern-library",
-  "version": "0.2.0",
+  "version": "0.1.46",
```

Comment:
I'm trying to version this properly: `major.minor.fixup`. I guess it doesn't __really__ matter though. What do you think? If we're not going to use it, should we just remove it?